pkg/buildpack/buildpack.go

```diff
@@ -48,22 +51,48 @@ type Blob interface {
 	Open() (io.ReadCloser, error)
 }
 
+type BlobOption func(*blobOption) error
+
+type blobOption struct {
+	flattened bool
+}
+
+func Flattened() BlobOption {
```

Comment:
Do we ever actually use this option? I only see it being called in tests...

Reply:
After merging the suggestions branch, we are not. I removed it